### DISCUSSION TITLE
Instantiate, parent - 생성 오브젝트를 다른 오브젝트 밑에 넣기

### DISCUSSION THREAD

**Discussion by janggf** [naverz/zepeto-studio-kor#518] [5. Q&A]
![캡처](https://user-images.githubusercontent.com/13530263/159452399-30c5a319-b6df-430e-84cf-7862a104af14.JPG)

cObject 가  cObject2 밑으로 가도록, 부모자식 형성하면 되는 간단한 일입니다
cObject 변수는 프로젝트창에 있는거 넣은 상태에요
cObject2 변수는 씬창에 있어요

var 변수...; c#하다가 오니까 기초에서 다막힙니다

**Comment by peterkangdev** [comment]
이렇게 써 보세요.
this.cObject.transform.SetParent(cObject2.transform);

**Comment by byeundo** [comment]
Unity Instantiate 함수는 5종류의 오버로드가 있는데
Declaration
public static Object Instantiate(Object original);
Declaration
public static Object Instantiate(Object original, Transform parent);
Declaration
public static Object Instantiate(Object original, Transform parent, bool instantiateInWorldSpace);
Declaration
public static Object Instantiate(Object original, Vector3 position, Quaternion rotation);
Declaration
public static Object Instantiate(Object original, Vector3 position, Quaternion rotation, Transform parent);

여기서 parent 인수로 받는 3종류 녀석중에 원하시는대로 골라쓰시면 대요~~

**Reply by byeundo** [reply]
예) Object.Instantiate(this.cObject,this.cObject2.transform);
